test/index.test.ts

```diff
@@ -201,4 +202,42 @@ describe('fp-ts-ramda', () => {
       { examples: [[0]] }
     );
   });
+  it('any', () => {
+    const odd = (n: number) => n % 2 !== 0;
+    const isCapitalized = (str: string) => (str.length ? /[A-Z]/.test(str[0]) : false);
+    fc.assert(
+      fc.property(
+        fc.array(fc.integer()),
+        as => eqBoolean.equals(R.any(odd, as), FR.any(odd, as)) && eqBoolean.equals(R.any(odd)(as), FR.any(odd)(as))
+      ),
+      { examples: [[[]]] }
+    );
+    fc.assert(
+      fc.property(
+        fc.array(
+          fc.record({
+            firstName: fc.string()
+          })
+        ),
+        as =>
+          eqBoolean.equals(
+            R.any(
+              R.pipe(
+                R.prop('firstName'),
+                isCapitalized
+              ),
+              as
+            ),
+            FR.any(
+              flow(
+                FR.prop('firstName'),
+                isCapitalized
+              ),
+              as
+            )
+          )
+      ),
+      { examples: [[[]]] }
```

Comment:
this is useless I think? Should be removed

Reply:
Are you referring to the second test or the examples? Ramda has a similar test to see if `any` works with objects but I would be inclined to agree that it is not very useful.  The examples is so that the test includes a test with an empty array.